src/main/msp/msp.c

```diff
@@ -2439,13 +2447,14 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         if (sbufBytesRemaining(src) >= 1) {
             rxConfigMutable()->fpvCamAngleDegrees = sbufReadU8(src);
         }
+        uint16_t rcSmoothingDerivativeCutoff;
         if (sbufBytesRemaining(src) >= 6) {
             // Added in MSP API 1.40
             rxConfigMutable()->rcInterpolationChannels = sbufReadU8(src);
 #if defined(USE_RC_SMOOTHING_FILTER)
             configRebootUpdateCheckU8(&rxConfigMutable()->rc_smoothing_type, sbufReadU8(src));
             configRebootUpdateCheckU8(&rxConfigMutable()->rc_smoothing_input_cutoff, sbufReadU8(src));
-            configRebootUpdateCheckU8(&rxConfigMutable()->rc_smoothing_derivative_cutoff, sbufReadU8(src));
+            rcSmoothingDerivativeCutoff = sbufReadU8(src);
```

Comment:
This should keep the `configRebootUpdateCheckU8()` to make it not break when used with legacy configuration tools that don't support the 16 bit `rc_smoothing_derivative_cutoff`.

Reply:
In the event of an older configurator being used the value gets held in memory and used in line 2488, it's the bottom of changes to this msp.c

If you call configRebootUpdateCheckU8 and then configRebootUpdateCheckU16 on the same config value its going to look like it always gets changed in certain situations.

Reply:
Ok, I see the problem. Splitting the `uint16_t` into two chunks is still inconsistent with how this is handled in other places in `msp.c`. The second read should just read the entire `uint16_t`, and replace the original 8 bit value.